spec/mailers/user_mailer_activity_spec.rb

```diff
@@ -0,0 +1,182 @@
+require 'spec_helper'
+
+
+RSpec.describe UserMailer::Activity do
```

Comment:
Breaking the mailer's helper method out into its own class let us separate concerns when testing. Here we can test that the `Activity` class build method is doing what it should. This allows the actual mailer spec to focus on its mailing duties.